### ISSUE TITLE
Add citation of paper

### ISSUE THREAD

**Issue by MansMeg** [rOpenGov/sweidnumbr#125]
Wait until paper is released

**Comment by pitkant** [comment]
I had the idea that maybe some changes in the current dev release (1.4.3) should be published in CRAN before publishing the paper? For example in #117 some harmonization in function names was done which I would like to refer in the paper to emphasise the shared function names in both packages. In current CRAN release 1.4.2 from March 2020 this change was not yet made. I will send the final version of the paper today for comments.

What kind of schedule did you have in mind for 1.5 version?

**Comment by MansMeg** [comment]
Yes. Please feel free to add those changes needed to the issue tracker for milestone 1.5 so I don't miss anything.

My intention is to release 1.5 when the paper is published. I guess there will be some review process so additional changes might be added. When we know when and when we will publish, I will fix 1.5 and also add the paper to the citation of the package (and in the startup message). So the time frame for 1.5. is:

1. Submit paper
2. If accepted, check what needs to be done in the package and add as issues in 1.5.
3. Fix v1.5 code on github
4. Publish paper and release 1.5. at the same time (ish).
